info/container.go

```diff
@@ -197,6 +197,8 @@ type CpuStats struct {
 		// Unit: nanoseconds
 		System uint64 `json:"system"`
 	} `json:"usage"`
+	// Smoothed average of number of runnable threads x 1000.
+	// We multiply by thousand to avoid using floats, but preserving precision.
 	Load int32 `json:"load"`
```

Comment:
Only concern I have here is that all of the above stats are instantaneous except this one. We should at least document over what time period or how this is smoothed. Idk if we should export smoothing over different intervals. As long as we do this one over a small enough one we should be okay.